vendor/github.com/docker/cli/cli/connhelper/commandconn/commandconn.go

```diff
@@ -37,7 +37,7 @@ func New(ctx context.Context, cmd string, args ...string) (net.Conn, error) {
 		c   commandConn
 		err error
 	)
-	c.cmd = exec.CommandContext(ctx, cmd, args...)
+	c.cmd = exec.Command(cmd, args...)
```

Comment:
Seeing a ton of failures in CI; initially was wondering if there's any relation to this change, but at a glance, I don't think that change should make a difference; all calls pass a fresh `context.Background()`;


```console
git grep 'commandconn.New'
client/connhelper/dockercontainer/dockercontainer.go:                   return commandconn.New(context.Background(), "docker", append(ctxFlags, []string{"exec", "-i", sp.Container, "buildctl", "dial-stdio"}...)...)
client/connhelper/kubepod/kubepod.go:                   return commandconn.New(context.Background(), "kubectl", "--context="+sp.Context, "--namespace="+sp.Namespace,
client/connhelper/podmancontainer/podmancontainer.go:                   return commandconn.New(context.Background(), "podman", "exec", "-i", sp.Container, "buildctl", "dial-stdio")
client/connhelper/ssh/ssh.go:                   return commandconn.New(context.Background(), "ssh", args...)
vendor/github.com/docker/cli/cli/connhelper/commandconn/commandconn.go://                               return commandconn.New(ctx, "docker", "exec", "-it", containerID, "docker", "system", "dial-stdio")
```